.lintr

```diff
@@ -0,0 +1,2 @@
+linters: with_defaults(line_length_linter(120), open_curly_linter = NULL)
+exclusions: list("inst/script.R","R/zzz.R"=1)
```

Comment:
What does `=1` mean? Also, should we exclude whole files or use the ` "# nolint"` or ` "# nolint start"` markers?

Reply:
`file.R=<linenumber>` so just exclude line 1, in this case due to on.load breaking snake_caee convention